### PR TITLE
[SD-3547] updated permission

### PR DESCRIPTION
**Pull request by mzorzella-simeon**
Added IdentityRiskEvent.ReadWrite.All and Policy.ReadWrite.IdentityProtection scopes

### CODE REVIEW

File: Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json [hunk 1]
```diff
@@ -16,9 +16,7 @@
     "preAuthorizedApplications": [
```
Comment (ashek-simeon) [inline Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json:16]: Would you be willing to reformat this file so it doesn't show as a change in a future PR?

File: Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json [hunk 2]
```diff
@@ -214,14 +214,6 @@
           "id": "ef5f7d5c-338f-44b0-86c3-351f46c8bb5f",
           "type": "Role"
         },
-        {
-          "id": "5eb59dd3-1da2-4329-8733-9dabdc435916",
```
Comment (ashek-simeon) [inline Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json:218]: Would you be willing to help me understand why we are removing this one?

File: Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json [hunk 3]
```diff
@@ -58,18 +58,6 @@
           "id": "e4aa47b9-9a69-4109-82ed-36ec70d85ff1",
           "type": "Scope"
         },
-        {
-          "id": "7b8a2d34-6b3f-4542-a343-54651608ad81",
```
Comment (ashek-simeon) [inline Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json:62]: Would you be willing to help me understand why we are removing this one?

Comment (mzorzella-simeon) [reply Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json:62]: We should not....checking why it got removed

File: application-scopes.json [hunk 4]
```diff
@@ -1,48 +1,49 @@
...
+  "059e5840-5353-4c68-b1da-666a033fc5e8": "Microsoft Graph - ThreatSubmissionPolicy.ReadWrite.All (Read and write all threat submission policies)",
   "edd3c878-b384-41fd-95ad-e7407dd775be": "Microsoft Graph - Policy.ReadWrite.Authorization (Read and write your organization's authorization policy)",
-  "46ca0847-7e6b-426e-9775-ea810a948356": "Microsoft Graph - Organization.ReadWrite.All (Read and write organization information)",
-  "662ed50a-ac44-4eef-ad86-62eed9be2a29": "Microsoft Graph - DeviceManagementServiceConfig.ReadWrite.All (Read and write Microsoft Intune configuration)",
-  "e60370c1-e451-437e-aa6e-d76df38e5f15": "Skype and Teams Tenant Admin API - user_impersonation (Access Microsoft Teams and Skype for Business data as the signed in user)",
+  "89fe6a52-be36-487e-b7d8-d061c450a026": "Microsoft Graph - Sites.ReadWrite.All (Edit or delete items in all site collections)",
   "4f5bc9c8-ea54-4772-973a-9ca119cb0409": "Microsoft Graph - Policy.ReadWrite.AccessReview (Read and write your organization's directory access review default policy)",
-  "ef4b5d93-3104-4664-9053-a5c49ab44218": "Microsoft Graph - Agreement.ReadWrite.All (Read and write all terms of use agreements)",
```
Comment (ashek-simeon) [inline application-scopes.json:13]: Would need to re-run the script to generate this file after the scopes are re-added, I think

**Review by ashek-simeon — APPROVED** [review]

**Review by Lance-Simeon — APPROVED** [review]

**Pull request merged**